mscore/pianoroll/pianolevelsfilter.h

```diff
@@ -46,78 +43,101 @@ class PianoLevelsFilter
       virtual bool isPerEvent() = 0;
       virtual int value(Staff* staff, Note* note, NoteEvent* evt) = 0;
       virtual void setValue(Staff* staff, Note* note, NoteEvent* evt, int value) = 0;
-};
+      };
 
 
 //---------------------------------------------------------
 //   PianoLevelFilterOnTime
 //---------------------------------------------------------
 
-class PianoLevelFilterOnTime : public PianoLevelsFilter
-{
+class PianoLevelFilterOnTime : public PianoLevelsFilter {
+      Q_DECLARE_TR_FUNCTIONS(PianoLevelFilterOnTime)
+
 public:
-      QString name() override { return "On Time"; }
+      QString name() override { return tr("Note on time", "amount note 'on time' is adjusted by"); }
```

Comment:
a) that (and the later strings) in this file) don't sow up on Transifex, so can't get translated. No idea why though :-(

b) that 2nd string is for disambiguation, to tell identical strings apart, it is not meant as a translators' hint, for that you'd need to use a comment, like this:

    QString name() override { /*: amount note 'on time' is adjusted by */ return tr("Note on time"); }

c) those strings are quite long, not fitting their pulldown menu too well.